android/src/main/java/com/RNFetchBlob/Utils/FileProvider.java

```diff
@@ -0,0 +1,4 @@
+package com.RNFetchBlob.Utils;
+
+public class FileProvider extends android.support.v4.content.FileProvider {
```

Comment:
I think this will now need to extend `androidx.core.content.FileProvider` instead of the old support library